python/setup.py

```diff
@@ -254,7 +254,7 @@ def get_option_from_sys_argv(option_str):
     os.environ['PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION'] = 'cpp'
 
   # Keep this list of dependencies in sync with tox.ini.
-  install_requires = ['six>=1.9']
+  install_requires = ['six>=1.9', 'setuptools']
```

Comment:
This setuptools dependency was just recently deleted in #7511 so I think something must be going wrong if it's getting reverted.